server/apps/microservices/src/processors/metadata-extraction.processor.ts

```diff
@@ -50,10 +50,11 @@ export class MetadataExtractionProcessor {
   async extractExifInfo(job: Job<IExifExtractionProcessor>) {
     try {
       const { asset, fileName, fileSize }: { asset: AssetEntity; fileName: string; fileSize: number } = job.data;
+      const exifData = await exifr.parse(asset.originalPath);
 
-      const fileBuffer = await readFile(asset.originalPath);
-
-      const exifData = await exifr.parse(fileBuffer);
+      if (!exifData) {
+        throw new Error(`can not fetch exif data from file ${asset.originalPath}`);
```

Comment:
as `ibra` reported, `Make` field is read from undefined, so `exifData` var may be `undefined` or `null`. In this case, I suggest not creating `exif` entity because if we still create `exif` entity, some null info will be shown on web/mobile.

Reply:
if we not create `exif` entity, something will be
<img width="211" alt="image" src="https://user-images.githubusercontent.com/2031404/187080974-2382abc4-4c69-4ab2-89a6-bfbbbc4892de.png">

Do we really need create `exif` entity in this case?

Reply:
There is no EXIF entity created if there is an error throw. 

The message indicates that there is no EXIF info available on that file

Reply:
yeah, but if `exifData` is `undefined` so how can you get exif info of file? Or in this case, I just skip without throwing any error, don't I?

Reply:
But I found we also have `try catch` encapsulation. If throw a error, I think we can read logs easily and found what cause make `exifData` undefined.

Reply:
I've just watched my logs. This error come from my screenshot imgs. Exifr can't extract meta data all of them. It means `exifData` is `undefined`

<img width="711" alt="image" src="https://user-images.githubusercontent.com/2031404/187083452-a076e420-109e-49bd-b267-44de6492530d.png">

Reply:
Agree, fair assetment